src/pytorch/auto_model.h

```diff
@@ -129,6 +129,25 @@ struct TorchAutoModel : torch::CustomClassHolder {
                 doSample, temperature, topK, topP, repetitionPenalty, stopWordsList_int32);
     }
 
+    torch::Tensor forward(torch::Tensor &inputIds) {
+        int batchSize = inputIds.size(0);
+        int seqLen = inputIds.size(1);
+        int vocabSize = model->getVocabSize();
+        int logitsN = batchSize * seqLen * vocabSize;
+
+        if (model->getRank() == 0) { input(inputIds); }
```

Comment:
For the original code, why 'input' method is not called 'setInput', input is more like a keyword, :(